packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -456,21 +469,40 @@ export default class CodeMode extends Component<Signature> {
                 {{if (eq this.fileView "browser") "file-browser"}}'
             >
               <header
+                class='file-view__header'
```

Comment:
Is this needed? I thought scoping would take care of it 😯

Reply:
yes, it takes care of it. I just found it easier to find the css via the class name